### PR TITLE
Export List Blocks

### PR DESCRIPTION
**Pull request by florencio-mazzoldi**
#### Overview
Now for list blocks it creates and downloads a zip file with a genbank file for each combination.
Also several enhancements/bug fixes for import/export

#### Type of change (bug fix, feature, docs, UI, etc.)
Both. Export lists + bug fixes.

#### Requirements

- [ ] Adds a test (for features + fixes)
- [ ] Docs updated (for features + fixes)

### CODE REVIEW

File: server/extensions/native/genbank/genbank_export.py [hunk 1]
```diff
@@ -158,9 +168,13 @@ def get_children_ids(block, allblocks):
     return children
 
 # Take a project structure and a list of all the current blocks, convert this data to a genbank file and store it
-# in filename.
-def project_to_genbank(filename, project, allblocks):
-    blocks = project["components"]
+# in filename. If you pass a construct in, only convert that particular construct.
+def project_to_genbank(filename, project, allblocks, construct_id=None):
```
Comment (maxbates) [inline server/extensions/native/genbank/genbank_export.py]: did you test if this works - clicking an item from NCBI search? Doing that passes `onlyConstructs=true` to the route and requests only constructs are passed. This is different than what you're addressing here (exporting only a particular construct from a project),  but may be impacted slightly?

File: server/extensions/native/genbank/index.js [hunk 2]
```diff
@@ -94,15 +94,11 @@ router.get('/export/blocks/:projectId/:blockIdList', permissionsMiddleware, (req
...
       };
 
-      console.log('Exporting 1');
-      console.log(JSON.stringify({ roll: partialRoll, constructId: construct.id }));
-
       exportConstruct({ roll: partialRoll, constructId: construct.id })
-        .then(fileContents => {
-          res.set({
-            'Content-Disposition': `attachment; filename="${roll.project.id}.fasta"`,
+        .then(resultFileName => {
```
Comment (maxbates) [inline server/extensions/native/genbank/index.js]: create a wrapper for this so we dont have lots of files left around?

**Comment by maxbates** [comment]
test for exporting a list block?

File: server/extensions/native/genbank/genbank_export.py (hunk 1 above)
Comment (florencio-mazzoldi) [inline server/extensions/native/genbank/genbank_export.py]: Tests pass for importing NCBI.

**Comment by florencio-mazzoldi** [comment]
@maxbates tests are in. Please re-review.

**Comment by maxbates** [comment]
can you merge with master again? the build is failing due to dependencies. Either you need to merge, or merged improperly. check `package.json`. The build should not fail (except maybe the github issue test if you are not in the PR environment)

**Comment by florencio-mazzoldi** [comment]
The build fails with this:
npm ERR! Linux 3.13.0-92-generic
npm ERR! argv "/home/travis/.nvm/versions/node/v4.5.0/bin/node" "/home/travis/.nvm/versions/node/v4.5.0/bin/npm" "install"
npm ERR! node v4.5.0
npm ERR! npm  v2.15.9
npm ERR! code EPEERINVALID
npm ERR! peerinvalid The package react@15.3.1 does not satisfy its siblings' peerDependencies requirements!
npm ERR! peerinvalid Peer react-addons-perf@15.3.1 wants react@^15.3.1
npm ERR! peerinvalid Peer react-addons-test-utils@15.3.1 wants react@^15.3.1
npm ERR! peerinvalid Peer react-dom@15.3.1 wants react@^15.3.1
npm ERR! peerinvalid Peer react-dropzone@3.6.0 wants react@^0.14.0 || ^15.0.0
npm ERR! peerinvalid Peer react-redux@4.4.5 wants react@^0.14.0 || ^15.0.0-0
npm ERR! peerinvalid Peer react-router@2.8.0 wants react@^0.14.0 || ^15.0.0
npm ERR! peerinvalid Peer redux-devtools-dock-monitor@1.1.1 wants react@^0.14.0 || ^15.0.0-0
npm ERR! peerinvalid Peer redux-devtools-log-monitor@1.0.11 wants react@^0.14.0 || ^15.0.0-0
npm ERR! peerinvalid Peer react-addons-clone-with-props@0.14.8 wants react@^0.14.8
npm ERR! peerinvalid Peer react-addons-css-transition-group@0.14.8 wants react@^0.14.8
npm ERR! peerinvalid Peer redbox-react@1.3.0 wants react@^0.14.0 || ^15.0.0
npm ERR! peerinvalid Peer redux-devtools@3.3.1 wants react@^0.14.0 || ^15.0.0-rc.1
npm ERR! peerinvalid Peer react-bootstrap@0.28.5 wants react@>=0.14.0
npm ERR! Please include the following file with any support request:
npm ERR!     /home/travis/build/autodesk-bionano/genome-designer/npm-debug.log

It doesn't seem to be related with my PR.

**Comment by maxbates** [comment]
failures are just github. this looks good.

**Pull request merged**